knowledge-base/objectdatasource-assembly-not-recognized-by-standalone-designer.md

```diff
@@ -0,0 +1,42 @@
+---
+title: ObjectDataSource Assembly Not Recognized by the Standalone Designer
+description: How to troubleshoot which assemblies to add to the Standalone designer folder when an ObjectDataSource is not property recognized
+type: troubleshooting
+page_title: 
+slug: objectdatasource-assembly-not-recognized-by-standalone-designer
+position: 
+tags: 
+ticketid: 1453347
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+When you use an ObjectDataSourcem, which DataSource has external dependencies, the Standalone designer may not recognize the main ObjectDataSource assembly even when it is properly added to the designer folder and registered in its configuration file.
+
+## Cause\Possible Cause(s)
+The reason is that the Standalone designer cannot find the external assemblies when trying to get the data source schema.
+
+## Solution
+You may use [Fuslogvw](https://docs.microsoft.com/en-us/dotnet/framework/tools/fuslogvw-exe-assembly-binding-log-viewer) to investigate the issue. The idea is to see what dependencies the designer tries to load and eventually which ones cannot be found. 
+Here we demonstrate the approach step-by-step with an ObjectDataSource that utilizes EntityFrameworkCore version 3.1.1.
```

Comment:
you may include a link for the object data source https://docs.telerik.com/reporting/objectdatasource

Reply:
Included in the start